### ISSUE TITLE
"Check for Updates" gives older "Latest" version than installed one.

### ISSUE THREAD

**Issue by v-karbovnichy** [mRemoteNG/mRemoteNG#1909]
![image](https://user-images.githubusercontent.com/7301634/100737555-c3ca1a80-33ed-11eb-9dc2-5738ec059df2.png)
I just wondered about if I have the latest version, saw auto-upgrade check, and found this:
* Installed version: 1.76.20.24615
* Latest version: 1.76.11.40527

I am sure I did not install this from Beta channel.
Anyway the latest release on GitHub is [1.76.20](https://github.com/mRemoteNG/mRemoteNG/releases/tag/v1.76.20)

## Expected Behavior
The latest version should be the same or greater than installed version.

## Current Behavior
The latest version is lower than installed version.

## Possible Solution
No solution, I don't know where this info comes from.
Also, as an improvement, I suggest mentioning what release channel is used when checking for updates.
And what release channel was used for the installed version (to understand was it taken from Beta or not)

## Steps to Reproduce (for bugs)

1. Open mRemoteNG
2. From the Help menu, click on the "Check for Updates" item
3. Wait until update check is done.
4. Compare the version numbers.

## Context
Was trying to find if the tool supports several windows in parallel. Navigated to settings, checked all menus, looking for some "Tile" option.

## Your Environment
* Version used: 1.76.20.24615
* Operating System and version (e.g. Windows 10 1709 x64): Windows 10 2004 x64 (10.0.19041.630)

**Comment by MarcF-B** [comment]
It is pulled from here:
https://mremoteng.org/update.txt

The info about the URL is from here:
https://github.com/mRemoteNG/mRemoteNG.github.io/blob/main/update.txt

Someone has to update the "update.txt" ...

**Comment by LIyoai** [comment]
#1853

**Comment by LIyoai** [comment]
Duplicate of #1853

**Comment by farosch** [comment]
Follow the above issue for follow-ups
